src/cowrie/output/datadog.py

```diff
@@ -0,0 +1,63 @@
+"""
+Simple Datadog HTTP logger.
+"""
+
+from __future__ import annotations
+
+import json
+import platform
+
+from io import BytesIO
+from twisted.internet import reactor
+from twisted.internet.ssl import ClientContextFactory
+from twisted.web import client, http_headers
+from twisted.web.client import FileBodyProducer
+
+import cowrie.core.output
+from cowrie.core.config import CowrieConfig
+
+
+class Output(cowrie.core.output.Output):
+    def start(self):
+        self.url = CowrieConfig.get("output_datadog", "url").encode("utf8")
+        self.api_key = CowrieConfig.get("output_datadog", "api_key").encode("utf8")
+        self.ddsource = CowrieConfig.get("output_datadog", "ddsource", fallback="cowrie")
+        self.ddtags = CowrieConfig.get("output_datadog", "ddtags", fallback="env:prod")
+        self.service = CowrieConfig.get("output_datadog", "service", fallback="honeypot")
```

Comment:
why do we use `ddsource` and `ddtags` but `service`?

Reply:
Based on official docs :)